src/ansys/hps/data_transfer/client/authenticate.py

```diff
@@ -49,34 +49,34 @@ def authenticate(
     verify: bool | str = True,
     **kwargs,
 ):
-    """Authenticates the user with a password or refresh token against the HPS authentication service.
+    """Authenticate the user with a password or refresh token against the HPS authentication service.
 
     If this method is successful, the response includes access and refresh tokens.
 
     Parameters
     ----------
 
-    url : str, optional
-        Base path for the server to call. The default is ``'https://localhost:8443/rep'``.
-    realm : str, optional
-        Keycloak realm. The default is ``'rep'``.
-    grant_type: str, optional
-        Authentication method. The default is ``'password'``.
-    scope : str, optional
-        String containing one or more requested scopes. The default is ``'openid'``.
-    client_id : str, optional
-        Client type. The default is ``'rep-cli'``.
-    client_secret : str, optional
-        Client secret. The default is ``None``.
-    username : str, optional
+    url : str, default: 'https://localhost:8443/hps'
+        Base path for the server to call.
+    realm : str, default: 'rep'
+        Name of the Keycloak realm.
+    grant_type: str, default: 'password'
+        Authentication method.
+    scope : str, default: 'openid'
+        String containing one or more requested scopes.
+    client_id : str, default: 'rep-cli'
+        Client type.
+    client_secret : str, default: None
+        Client secret.
+    username : str
         Username.
-    password : str, optional.
+    password : str
         Password.
-    refresh_token : str, optional
+    refresh_token : str
         Refresh token.
-    timeout : float, optional
-        Timeout in seconds. The default is ``10.0``.
-    verify: Union[bool, str], optional
+    timeout : float, default: 10.0
+        Timeout in seconds.
+    verify: Union[bool, str]
```

Comment:
The Returns section for this class is formatted oddly in the documentation. I removed the colon after Returns but pre-commit "fixed" it since it said the removal was an error.

Reply:
The autoapi doc made it look weird. It makes sense to take it out. Addressed in the latest commit. Thanks